scripts/generate_usb_vif/generate_vif.py

```diff
@@ -17,16 +18,14 @@
 SCRIPTS_DIR = os.path.join(os.path.dirname(__file__), "..")
 sys.path.insert(0, os.path.join(SCRIPTS_DIR, 'dts', 'python-devicetree', 'src'))
 
-from devicetree import edtlib
-
-
 def main():
+    global edtlib
+
     args = parse_args()
-    try:
-        with open(args.edt_pickle, 'rb') as f:
-            edt = pickle.load(f)
-    except edtlib.EDTError as err:
-        sys.exit(f"devicetree error: {err}")
+    with open(args.edt_pickle, 'rb') as f:
+        edt = pickle.load(f)
```

Comment:
This can still raise an error through right, should this be caught?

Reply:
Yes, it can raise an error, but `generate_vif` will just halt in that case.  I don't see a need catching any specific errors.